<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowserDataLog">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>371</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConectado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>***********</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEditInputServerIP">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Insira o IP do Servidor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonConectar">
         <property name="text">
          <string>Conectar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonDesconectar">
         <property name="text">
          <string>Desconectar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QLCDNumber" name="lcdNumberMinimo"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Valor Mínimo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSliderMinimo">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QLCDNumber" name="lcdNumberMaximo"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Valor Máximo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSliderMaximo">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tempo de Envio</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="horizontalSliderTempo">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonIniciar">
       <property name="text">
        <string>Iniciar Envio</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonParar">
       <property name="text">
        <string>Parar Envio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMinimo</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumberMinimo</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMaximo</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumberMaximo</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTempo</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMinimo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMinimo</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMaximo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMaximo</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTempo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
